posix.mak

```diff
@@ -29,7 +29,12 @@ INSTALL_DIR=../install
 DOCDIR=doc
 IMPDIR=import
 
-OPTIONAL_PIC:=$(if $(PIC),-fPIC,)
+# -fPIC is enabled by default and can be disabled with PIC=0
+ifneq ($(PIC),0)
+    DEFAULT_PIC:=-fPIC
```

Comment:
I don't care _that_ much either ;)
But someone else reading this in the future might get confused, because it's not really a default if it can be changed on the command line.